src/LayerEditor/ui/diagram_editor/graphics_items/gs_point.py

```diff
@@ -55,8 +56,8 @@ def __init__(self, pt, block):
     def shape_id(self):
         return self.pt.id
 
-    def init_regions(self, block):
-        for layer in block.layers:
+    def init_regions(self):
```

Comment:
This is an operation for consitency of the model and should be in the model classes, probably in BlockModel.

Reply:
Yes, it already was moved there